gallery/src/demos/demo-hui-entities-card.ts

```diff
@@ -191,6 +236,7 @@ class DemoEntities extends PolymerElement {
   public ready() {
     super.ready();
     const hass = provideHass(this.$.demos);
+    hass.updateTranslations(null, "en");
```

Comment:
Feels weird that we need to call this in every demo. Does it make sense to add this to `provideHass` ? (although I guess `providehass` now works without relying on any other files)

Reply:
We can't add it to `provideHass` as the demo also uses that.

I would like to generate hass a level higher and pass that to the demos instead of generating a hass in every demo. Might lose a little flexibility in the tests, although if really needed you could still create your own hass... But that is for a different PR.